src/features/bakery/Bakery.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Modal } from "react-bootstrap";
+
+import bakery from "assets/buildings/bakery.png";
+import soup from "assets/icons/bakery.png";
+
+import { Crafting } from "./components/Crafting";
+import { Action } from "components/ui/Action";
+import { GRID_WIDTH_PX } from "features/game/lib/constants";
+
+export const Bakery: React.FC = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  return (
+    <div
+      className="z-10 absolute"
+      // TODO some sort of coordinate system
+      style={{
+        width: `${GRID_WIDTH_PX * 100}px`,
```

Comment:
<img width="713" alt="Screen Shot 2022-01-25 at 9 31 57 am" src="https://user-images.githubusercontent.com/11745561/150977555-2956a123-5176-49a1-b24f-b9de85689090.png">

This translates roughly to 4200px, I would suggest making this smaller to avoid any unwanted styling overflow